src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -0,0 +1,170 @@
+import type { CSSResultGroup } from "lit";
+import { css, html, LitElement, nothing } from "lit";
+import { customElement, state } from "lit/decorators";
+import { mdiEye, mdiEyeOff } from "@mdi/js";
+import { formatDateTime } from "../../../../common/datetime/format_date_time";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import "../../../../components/ha-alert";
+import "../../../../components/ha-button";
+import "../../../../components/ha-icon-button";
+import { createCloseHeading } from "../../../../components/ha-dialog";
+import { haStyleDialog } from "../../../../resources/styles";
+import type { HomeAssistant } from "../../../../types";
+import type { CloudAlreadyConnectedParams as CloudAlreadyConnectedDialogParams } from "./show-dialog-cloud-already-connected";
+import { obfuscateUrl } from "../../../../util/url";
+
+@customElement("dialog-cloud-already-connected")
+class DialogCloudAlreadyConnected extends LitElement {
+  public hass!: HomeAssistant;
+
+  @state() private _params?: CloudAlreadyConnectedDialogParams;
+
+  @state() private _obfuscateIp = true;
+
+  public showDialog(params: CloudAlreadyConnectedDialogParams) {
+    this._params = params;
+  }
+
+  public closeDialog() {
+    this._params?.closeDialog();
+    this._params = undefined;
+    this._obfuscateIp = true;
+    fireEvent(this, "dialog-closed", { dialog: this.localName });
+  }
+
+  protected render() {
+    if (!this._params) {
+      return nothing;
+    }
+    const { details } = this._params;
+
+    return html`
+      <ha-dialog
+        open
+        @closed=${this.closeDialog}
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize(
+            "ui.panel.config.cloud.dialog_already_connected.heading"
+          )
+        )}
+      >
+        <div>
+          <p>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.description"
+            )}
+          </p>
+          <b>
+            ${this.hass.localize(
+              "ui.panel.config.cloud.dialog_already_connected.other_home_assistant"
+            )}
+          </b>
+          <div class="instance-details">
+            <div class="instance-detail">
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.ip_address"
+                )}:
+              </p>
+              <div class="obfuscated">
+                <p>
+                  ${this._obfuscateIp
+                    ? obfuscateUrl(details.remote_ip_address)
+                    : details.remote_ip_address}
+                </p>
+
+                <ha-icon-button
+                  class="toggle-unmasked-url"
+                  toggles
```

Comment:
What is this property for?

Reply:
https://github.com/home-assistant/frontend/blob/dev/src/panels/config/network/ha-config-url-form.ts#L156
You tell me, I assume it is for a button that toggles.

Reply:
It seems to be a relict from the old polymer `paper-icon-button`, it has no function.